extensions/pkg/controller/backupbucket/actuator.go

```diff
@@ -12,10 +12,18 @@ import (
 	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
 )
 
-// Actuator acts upon BackupBucket resources.
+// Actuator acts upon [extensionsv1alpha1.BackupBucket] resources.
```

Comment:
```suggestion
// Actuator acts upon extensionsv1alpha1.BackupBucket resources.
```

Are you having a strong preference to use square brackets? FMPOV, we shouldn't need them and haven't do so in the past. It's probably hard find all places to change them consistently.

Reply:
> Are you having a strong preference to use square brackets? FMPOV, we shouldn't need them and haven't do so in the past.

No strong preference, that's just the way you create links in doc comments :)

- https://go.dev/doc/comment#package
- https://go.dev/doc/comment#links

Here's how the documentation looks without the links.

<img width="1293" height="150" alt="Screenshot 2025-12-16 at 12 40 27" src="https://github.com/user-attachments/assets/3cb5c9cf-7684-49ba-a992-683e8c5d02d3" />

And this is how it looks with the link.
<img width="1284" height="145" alt="Screenshot 2025-12-16 at 12 39 14" src="https://github.com/user-attachments/assets/de18334a-7c25-4f1b-8a97-f8603b54adf8" />

This makes the documentation better, but also LSP knows how to follow such links as well. This also lets you jump between symbol definitions, even from comments.

My goal is not go over every single doc comment in Gardener, but focus on the extensions API doc for now, we can follow up and fix the rest as we go, but for now I'd prefer keeping the links, because of the additional benefits they provide.

Reply:
Nice, I wasn't aware about this - shame on me. Thanks for clarifying 🙂